.azure-pipelines.yml

```diff
@@ -47,13 +47,18 @@ jobs:
         env:
           GH_TOKEN: $(LENS_IDE_GH_TOKEN)
         displayName: Customize config
-      
+
       - script: make node_modules
         displayName: Install dependencies
-      
+
+      - base: |
```

Comment:
Probably would make sense to do this within `make build`:

```Makefile
build: node_modules binaries/client
	yarn run npm:fix-build-version
	$(MAKE) build-extensions -B
	yarn run compile
ifeq "$(DETECTED_OS)" "Windows"
	rm -rf node_modules/win-ca/pem
	yarn run electron-builder --publish onTag --x64 --ia32
else
	yarn run electron-builder --publish onTag
endif
```

Reply:
Sounds good will change